src/Tickets/Commerce/Hooks.php

```diff
@@ -416,4 +420,17 @@ public function add_display_post_states( $post_states, $post ) {
 		return $post_states;
 	}
 
+	/**
+	 * Handle saving of Ticket provider meta data.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $meta_key   Metadata key.
+	 * @param mixed  $meta_value Metadata value.
+	 *
+	 * @return string
+	 */
+	public function handle_provider_meta( $meta_value, $meta_key ) {
```

Comment:
Since this is a filter, we should prefix it. `filter_modify_sanitization_provider_meta`